### PR TITLE
added landing screens for landing page and home page

### PR DESCRIPTION
**Pull request by SHIKHASINGH1506**
1. Added screens for landing page and home page
2. links are not added at all the places yet

### CODE REVIEW

**Review by vishalg8454 — APPROVED** [review]
All good to merge!

File: index.html [hunk 1]
```diff
@@ -8,6 +8,50 @@
...
+                    id="sidebar-toggler"
+                    viewBox="0 0 100 40"
+                    width="40"
+                    height="40"
+                >
+                    <rect width="60" height="10"></rect>
+                    <rect y="20" width="60" height="10"></rect>
+                    <rect y="40" width="60" height="10"></rect>
+                </svg>
+                <a class="brand-logo mx-2" href="/">Noter</a>
```
Comment (rahulyadav139) [inline index.html:25]: suggestion: use Link component instead of anchor tag.

File: index.html [hunk 2]
```diff
@@ -8,6 +8,50 @@
...
+            <div class="d-flex">
+                <i class="fa-solid fa-arrow-right-from-bracket"></i>
+            </div>
+            </div>
+        </nav>
+    </header>
+    <div class="landing-wrapper p-4">
+        <div class="landing-content-wrapper d-flex justify-between ">
+          <div class="hero-img-wrapper">
+            <img src='./assets/images/hero-img.svg' alt="" class="responsive-img" />
```
Comment (rahulyadav139) [inline index.html:36]: suggestion: alt tag should not be empty.

**Review by rahulyadav139 — APPROVED** [review]
Codes are looking great and readable. great work.

**Pull request merged**